tests/Avalonia.Markup.Xaml.UnitTests/Xaml/ControlTemplateTests.cs

```diff
@@ -41,7 +41,7 @@ public void StyledProperties_Should_Be_Set_In_The_ControlTemplate()
 
                 window.ApplyTemplate();
                 button.ApplyTemplate();
-                var listBoxHierarchyLine = button.GetVisualChildren().ElementAt(0) as ListBoxHierachyLine;
+                var listBoxHierarchyLine = button.GetVisualChildren().ElementAt(0) as ListBoxHierarchyLine;
```

Comment:
ListBoxHierarchyLine was renamed in the C# code, but wasn't in the XAML code. This test is failing.

Reply:
good catch. fixed